Guide di Sopravvivenza/vocabolario.md

```diff
@@ -0,0 +1,41 @@
+# Vocabolario
+Di seguito alcune definizioni di termini di carattere universitario, che potresti sentire in giro e potrebbero generare confusione, ma che non hai il coraggio di chiedere. 
+Pro tip: Impara a googlare 
+Pro Pro tip: Se non sai cosa significa qualcosa, chiedi! Non c'è nessuna vergogna, dobbiamo tutti imparare da qualche parte.
+
+
+### A: 
+- **AlmaLaurea**: Portale online che raccoglie i curricula degli studenti universitarie. Consulta le informazioni sul corso di laurea [Triennale](https://www2.almalaurea.it/cgi-php/universita/statistiche/visualizza.php?anno=2022&corstipo=tutti&ateneo=70031&facolta=1265&gruppo=tutti&livello=1&area4=tutti&pa=70031&classe=tutti&postcorso=tutti&isstella=0&isstella=0&presiui=tutti&disaggregazione=&LANG=it&CONFIG=profilo) e [Magistrale](https://www2.almalaurea.it/cgi-php/universita/statistiche/visualizza.php?anno=2022&corstipo=tutti&ateneo=70031&facolta=1265&gruppo=tutti&livello=2&area4=tutti&pa=70031&classe=tutti&postcorso=tutti&isstella=0&isstella=0&presiui=tutti&disaggregazione=&LANG=it&CONFIG=profilo).
+- **Appello**: Data in cui si svolge un esame. 
+### C:
+- **CFU**: Crediti Formativi Universitari. Sono una misura del carico di lavoro richiesto per superare un esame o un corso. Un CFU corrisponde a 25 ore di lavoro, comprensive di lezioni frontali, esercitazioni, studio individuale e preparazione all'esame.
+- **Classe di Laurea**: Codice identificativo del corso di laurea (es. L-31 per informatica Triennale e LM-18 per informatica Magistrale).
+- **Corso di Laurea/ di Studio**: Tipologia di laura che si può conseguire. Nel nostro caso informatica. 
+- **Corso**: Insieme di lezioni che trattano un argomento specifico.
+### E:
+- **EDISU** : Ente regionale per il diritto allo studio universitario del Piemonte. Offre borse di studio, servizi abitativi e ristorazione a studenti universitari.
+- **Esame**: Prova scritta o orale che verifica la conoscenza di un argomento. In alcuni casi suddivisa in paraziali o esoneri. Potrebbe richiedere lo sviluppo di un progetto. Gli esami possono essere superati con voto in trentesimi (da 18 a 30 e lode).
+- **ESSE3**: Sistema informatico dell'Università di Torino che permette di iscriverti agli appelli.
```

Comment:
_Sistema informatico che l'Università di Torino usa [...]_

Aggiungerei questa precisazione perchè in realtà è un sistema inter-universitario